archlinux/license.md

```diff
@@ -0,0 +1 @@
+The Docker image creation scripts contained under the repository %%REPO%% are licensed under GPLv3. However, all applications contained in each tag are licensed under their specific licenses.
```

Comment:
Just to be clear, this content is going to have https://github.com/docker-library/docs/blob/73b89b28e5133c925a8dd6104d16ebaf01f5fc46/.template-helpers/license-common.md appended after it (in case that changes your intended wording here).

Reply:
Ah, well yeah that's a little redundant then. Let me reword with this into consideration.